Gemfile

```diff
@@ -24,11 +24,13 @@ gem 'rails_autolink'
 # Please don't update hoptoad_notifier to airbrake.
 # It's for internal use only, and we monkeypatch certain methods
 gem 'hoptoad_notifier', "~> 2.4"
-gem 'draper', :require => false
+gem 'draper'
 
 gem 'errbit_plugin', github: 'errbit/errbit_plugin'
 gem 'errbit_github_plugin', github: 'errbit/errbit_github_plugin'
 
+gem 'dotenv-deployment'
```

Comment:
I'm on board with this approach. I was just wondering if we want people to be able to do all this stuff:
https://github.com/bkeepers/dotenv-deployment/blob/master/lib/dotenv/deployment.rb#L8-L14

It seems pretty weird, but I guess its ok. I'll just not use that feature.